cmake/templates/BoostConfig.cmake.in

```diff
@@ -34,7 +34,6 @@ endfunction(hunter_add_rt_library)
 
 hunter_status_debug("Boost find config")
 
-set(Boost_USE_STATIC_LIBS ON)
```

Comment:
Should not be removed, use `if(@Boost_USE_STATIC_LIBS@)`. As you may see using same variable for the doing configure_file of template and variable on user's side is confusing.